.travis.yml

```diff
@@ -136,7 +136,9 @@ jobs:
             php: 7.3
             name: Test Coverage
             script:
-                - phpdbg -qrr -d memory_limit=-1 vendor/bin/phpunit --coverage-clover coverage.xml
+                # https://kizu514.com/blog/pcov-is-better-than-phpdbg-and-xdebug-for-code-coverage/
+                - pecl install pcov
+                - vendor/bin/phpunit --configuration phpunit.xml --coverage-clover coverage.xml
```

Comment:
Btw, why `--configuration phpunit.xml`? It's default value

Reply:
Copied from the article :)